addons/controllers/packageinstallstatus_controller.go

```diff
@@ -205,12 +205,13 @@ func (r *PackageInstallStatusReconciler) reconcile(clusterClient client.Client,
 
 	var errorList []error
 
+	patchHelper, err := clusterapipatchutil.NewHelper(clusterBootstrap, r.Client)
+	if err != nil {
+		errorList = append(errorList, errors.Wrap(err, "error patching ClusterBootstrapStatus"))
+		retErr = kerrors.NewAggregate(errorList)
```

Comment:
@vijaykatam This block needed to be outside of the defer, the tests were failing sue to this change.